### PR TITLE
Time table

### LINKED ISSUE #50: Make Timetable to optionally consider a visitor's visits.
Let's make possible to produce a customised timetable for a Visitor. 

The result should be a calendar that have all group's activities together with a Visitor's activities (indicate somehow what activities are subscribed).

### PR DESCRIPTION
**Pull request by Lehersun**
Created new function in TimeTable:class to make possible customise timetable for a Visitor.

### CODE REVIEW

**Comment by Lehersun** [comment]
For better recognizing I will merge it with master after previous commit will be merged

**Review by Magomogo — CHANGES_REQUESTED** [review]
Please actualize this branch.

**Pull request merged**
